mobile/lib/ui/settings/TEMP/lock_screen_option_confirm_pin.dart

```diff
@@ -0,0 +1,386 @@
+import "package:flutter/material.dart";
+import "package:flutter/services.dart";
+import "package:photos/core/configuration.dart";
+import "package:photos/theme/ente_theme.dart";
+import "package:photos/ui/components/buttons/icon_button_widget.dart";
+import "package:pinput/pinput.dart";
+
+class LockScreenOptionConfirmPin extends StatefulWidget {
+  const LockScreenOptionConfirmPin({super.key, required this.pin});
+  final String pin;
+  @override
+  State<LockScreenOptionConfirmPin> createState() =>
+      _LockScreenOptionConfirmPinState();
+}
+
+class _LockScreenOptionConfirmPinState
+    extends State<LockScreenOptionConfirmPin> {
+  final _confirmPinController = TextEditingController(text: null);
+  final Configuration _configuration = Configuration.instance;
+  Key _pinputKey = UniqueKey();
+  final _pinPutDecoration = PinTheme(
+    height: 48,
+    width: 48,
+    decoration: BoxDecoration(
+      border: Border.all(color: const Color.fromRGBO(45, 194, 98, 1.0)),
+      borderRadius: BorderRadius.circular(15.0),
+    ),
+  );
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    super.dispose();
+    _confirmPinController.dispose();
+  }
+
+  String _pin = "";
```

Comment:
Do we need this variable? I haven't checked all uses of `_pin`, but wouldn't just using `_confirmPinController.text` instead be enough?